src/main.py

```diff
@@ -1,13 +1,14 @@
 # setting this file up for an initial entry point for docker set up.
-import psycopg2
-
-conn = psycopg2.connect(
-    host="localhost",
-    port=5432,
-    database="appdb",
-    user="appuser",
-    password="apppassword"
-)
+import mysql.connector
+from mysql.connector import Error
+
+conn = mysql.connector.connect(
+        host="db",          # matches the service name in docker-compose.yml
```

Comment:
```suggestion
        host="localhost",          # matches the service name in docker-compose.yml
```
when it set to db it causes an error, which is that it cannot connect to mysql docker instance on the local machine, please make this change